clang/include/clang/Basic/TargetOptions.h

```diff
@@ -88,7 +88,7 @@ class TargetOptions {
     COV_5 = 500,
   };
   /// \brief Code object version for AMDGPU.
-  CodeObjectVersionKind CodeObjectVersion = CodeObjectVersionKind::COV_None;
+  CodeObjectVersionKind CodeObjectVersion = CodeObjectVersionKind::COV_4;
```

Comment:
Is this the default?

Reply:
Apologies. This change is not relevant to this PR. Removed it.